### PR TITLE
Address editorial issues

### LINKED ISSUE #21: IANA considerations section needed
since this spec is defining a new HTTP header field, it should have an IANA considerations section, which registers the defined with IANA in the header field registry.

### PR DESCRIPTION
**Pull request by yoavweiss**
This closes #21, #23 and #44

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -84,7 +84,7 @@ The ABNF (Augmented Backus-Naur Form) syntax for the `Device-Memory` header fiel
 <h3 dfn>Computing Device Memory Value</h3>
 The value is calculated by using the actual device memory in MiB then rounding it to the nearest number where only the most signicant bit can be set and the rest are zeros (nearest power of two). Then dividing that number by 1024.0 to get the value in GiB.
 
-An upper bound and a lower bound should be set on the list of values.
+An upper bound and a lower bound SHOULD be set on the list of values.
```
Comment (clelland) [inline index.bs]: Uppercasing RFC2119 keywords isn't strictly necessary; per RFC8174's clarification

(If you wanted, we could also include a section like the one in INFRA: https://infra.spec.whatwg.org/#conformance), but I don't think it's required.

File: index.bs [hunk 2]
```diff
@@ -131,9 +131,26 @@ Device identification and classification (e.g. device type and class) based on a
 
 To reduce fingerprinting risk, reported value is rounded to single most significant bit, as opposed to reporting the exact value. In addition, an upper and lower bound is placed on the reported values.
 
+IANA considerations {#iana}
+===================
+This document defines the `Device-Memory` HTTP request header field, and register them in the permanent message header field registery ([[RFC3864]]).
```
Comment (clelland) [inline index.bs]: nit: "registers"

File: index.bs [hunk 3]
```diff
@@ -131,9 +131,26 @@ Device identification and classification (e.g. device type and class) based on a
...
+
+Device-Memory Header field {#iana-device-memory}
+---------------------------------
+Header field name: Device-Memory
```
Comment (clelland) [inline index.bs]: Should this use bikeshed's definition list syntax?

```
:Header field name
:: `Device-Memory`
... etc
```

File: index.bs [hunk 4]
```diff
@@ -131,9 +131,26 @@ Device identification and classification (e.g. device type and class) based on a
...
+
+Applicable protocol: http
+
+Status: standard
+
+Author/Change controller: IETF
+
+Specification document: this specification ([[#sec-device-memory-client-hint-header]])
```
Comment (clelland) [inline index.bs]: Optional -- If this is going to be exported into the actual registry, then we can put `sec-device-memory-client-hint-header` into a `Required IDs` section in the top metadata, to ensure that the links don't break if the document changes.

File: index.bs (hunk 1 above)
Comment (yoavweiss) [reply index.bs]: Lowercased

File: index.bs (hunk 2 above)
Comment (yoavweiss) [reply index.bs]: done!

File: index.bs (hunk 4 above)
Comment (yoavweiss) [reply index.bs]: Done!

File: index.bs (hunk 3 above)
Comment (yoavweiss) [reply index.bs]: Sure!

**Review by clelland — APPROVED** [review]
Looks great, thanks!

**Pull request merged**